paparazzi/paparazzi-gradle-plugin/src/test/projects/compose-a11y-wear/src/test/java/app/cash/paparazzi/plugin/test/ComposeA11yWearTest.kt

```diff
@@ -0,0 +1,34 @@
+package app.cash.paparazzi.plugin.test
+
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.Paparazzi.Companion.determineHandler
+import app.cash.paparazzi.accessibility.A11ySnapshotHandler
+import app.cash.paparazzi.accessibility.ComposeA11yExtension
+import org.junit.Rule
+import org.junit.Test
+
+class ComposeA11yWearTest {
+  private val composeA11yExtension = ComposeA11yExtension()
+  private val maxPercentDifference = 1.0
```

Comment:
nit: just upping the max difference here isn't a good solution to the images not being exact on different arch/platforms.

Reply:
I'll remove those things that cause differences from the default implementation, but allow enough extension to allow me as a user of this library, to put them back in my app code.